### PR TITLE
Add issue templates for bugs and feature requests

### PR DESCRIPTION
**Pull request by philipqueen**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```
Comment (jonmatthis) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]:
```suggestion
A clear and concise description of what you expected to happen vs what actually happened.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE REQUEST]"
+labels: feature
+assignees: ''
+
+---
+
```
Comment (jonmatthis) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]: We should ask them something about their background, as it will help us understand their request

**Review by jonmatthis — CHANGES_REQUESTED** [review]
minor tweaks

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (philipqueen) [reply .github/ISSUE_TEMPLATE/feature_request.md:9]: I've added this in the format of "What is the use case for this feature?", with some help text pointing towards their background.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen vs what actually happened.
+
+**Log**
+Please include the log information for the session where you encountered this error. Logs can be found in `freemocap_data/logs_info_and_settings/logs` or copy and pasted from the terminal. Include as much context as possible so we can see upstream errors.
```
Comment (jonmatthis) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
Please include the log information for the session where you encountered this error. Logs can be found in `[path_to_your_user_directory]/freemocap_data/logs_info_and_settings/logs` or copy and pasted from the terminal. Include as much context as possible so we can see upstream errors.
```

Add a hint for the full path to the `freemocap_data` folder. 

Some True N00bs might still be confused by that, but its better than nothing.

**Review by jonmatthis — APPROVED** [review]
Approved after making the little change to the path (feel free to adapt my suggestion to make it less ugly, follow your heart)

**Pull request merged**